src/app/components/DappBanner/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react'
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import Box from '@mui/material/Box'
+import { useTranslation } from 'react-i18next'
+import { SearchScope } from '../../../types/searchScope'
+import { getDappForAddress } from '../../config/dapps'
+import Button from '@mui/material/Button'
+import { useScreenSize } from '../../hooks/useScreensize'
+
+export const DappBanner: FC<{ scope: SearchScope; address: string | undefined }> = ({ scope, address }) => {
+  const { t } = useTranslation()
+  const { isMobile } = useScreenSize()
+
+  if (!address) {
+    return null
+  }
+
+  const dApp = getDappForAddress(t, scope.network, scope.layer, address)
+
+  return (
+    !!dApp && (
+      <Card
+        sx={{
+          backgroundColor: COLORS.brandMedium,
+        }}
+        style={{
```

Comment:
Is there a reason for using `sx` for one css rule and `style` for another one?

Reply:
No, it was a leftover from an earlier version when I was experimenting something which couldn't be done as part of `sx` because of the special interpretation of the values there. Now I folded everything back to sx.)